images/app2appsuggestion.puml

```diff
@@ -0,0 +1,26 @@
+@startuml component
+actor merchantApp
+actor merchantBackend
+actor merchantAppControlledBrowser
+actor vippsApp
+actor vippsBackend
```

Comment:
Feels wrong to have a stick-person for every/any? step. The user isn't even represented by any step just the action between them.